x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/eql/trial_license_complete_tier/eql_alert_suppression.ts

```diff
@@ -3181,24 +3179,26 @@ export default ({ getService }: FtrProviderContext) => {
         });
       });
 
-      it('does not suppress alerts outside of duration when query with 3 sequences', async () => {
+      it.only('does not suppress alerts outside of duration when query with 3 sequences', async () => {
         const id = uuidv4();
-        const dateNow = Date.now();
-        const timestampSequenceEvent1 = new Date(dateNow - 5000).toISOString();
-        const timestampSequenceEvent2 = new Date(dateNow - 5500).toISOString();
-        const timestampSequenceEvent3 = new Date(dateNow - 5800).toISOString();
 
-        const firstSequenceEvent = {
-          id,
-          '@timestamp': timestampSequenceEvent1,
-          host: {
-            name: 'host-a',
-          },
-        };
+        const SECOND = 1000;
+        const dateNow = Date.now();
+        const timestamp1 = new Date(dateNow - 50 * SECOND).toISOString();
```

Comment:
It would be helpful to include some comment about the relationship between these timestamps and the rule's execution; just looking at this list it's unclear what role each individual timestamp has.

Maybe something like: 

> these first three timestamps will be queried during the first rule execution, due to the rule's 40s lookback.

I bring this up because looking at these times, I don't understand why the docs from 3, 4, and 5 seconds ago _wouldn't_ be found in the first rule execution.